### ISSUE TITLE
sleap-label raises TypeError: attrs() got an unexpected keyword argument 'eq'

### ISSUE THREAD

**Issue by jgraving** [murthylab/sleap#346]
Congrats! SLEAP looks amazing. Tried installing on my MacBook but immediately ran into a snag:
```bash
Jakes-MacBook-Pro:~ jake$ sleap-label
Traceback (most recent call last):
  File "/usr/local/bin/sleap-label", line 5, in <module>
    from sleap.gui.app import main
  File "/usr/local/lib/python3.7/site-packages/sleap/__init__.py", line 12, in <module>
    import sleap.nn
  File "/usr/local/lib/python3.7/site-packages/sleap/nn/__init__.py", line 7, in <module>
    import sleap.nn.inference
  File "/usr/local/lib/python3.7/site-packages/sleap/nn/inference.py", line 17, in <module>
    from sleap.nn.tracking import Tracker, run_tracker
  File "/usr/local/lib/python3.7/site-packages/sleap/nn/tracking.py", line 191, in <module>
    @attr.s(eq=False, slots=True, auto_attribs=True)
TypeError: attrs() got an unexpected keyword argument 'eq'
```

Also get the same error with `import sleap`. Here is the full `pip install` output if that's relevant:
```bash
Jakes-MacBook-Pro:~ jake$ python3 -m pip install sleap
Collecting sleap
  Downloading https://files.pythonhosted.org/packages/40/7a/3a4ec7acaf95023df04501c3d46fb01d3e8f51329c77d1346d0ec41c3ce5/sleap-1.0.0-py3-none-any.whl (32.4MB)
     |████████████████████████████████| 32.4MB 3.5MB/s 
Collecting seaborn
  Downloading https://files.pythonhosted.org/packages/c7/e6/54aaaafd0b87f51dfba92ba73da94151aa3bc179e5fe88fc5dfb3038e860/seaborn-0.10.1-py3-none-any.whl (215kB)
     |████████████████████████████████| 225kB 7.4MB/s 
Collecting jsmin
  Downloading https://files.pythonhosted.org/packages/17/73/615d1267a82ed26cd7c124108c3c61169d8e40c36d393883eaee3a561852/jsmin-2.2.2.tar.gz
Collecting scikit-video
  Downloading https://files.pythonhosted.org/packages/b1/a6/c69cad508139a342810ae46e946ebb3256aa6e42f690d901bb68f50582e3/scikit_video-1.1.11-py2.py3-none-any.whl (2.3MB)
     |████████████████████████████████| 2.3MB 31.4MB/s 
Collecting tensorflow==2.1
  Using cached https://files.pythonhosted.org/packages/35/55/a0dbd642e68e68f3e309d1413abdc0a7aa7e1534c79c0fc2501defb864ac/tensorflow-2.1.0-cp37-cp37m-macosx_10_11_x86_64.whl
Requirement already satisfied: scikit-image in ./Library/Python/3.7/lib/python/site-packages (from sleap) (0.15.0)
Requirement already satisfied: h5py in ./Library/Python/3.7/lib/python/site-packages (from sleap) (2.9.0)
Collecting opencv-python-headless==4.2.0.34
  Downloading https://files.pythonhosted.org/packages/c6/79/c2a8abc315fb8ea0c3cebd8cf9b0d03d9f1dca0b957dce26742002b0c259/opencv_python_headless-4.2.0.34-cp37-cp37m-macosx_10_9_x86_64.whl (40.2MB)
     |████████████████████████████████| 40.2MB 17.0MB/s 
Collecting qimage2ndarray==1.8
  Downloading https://files.pythonhosted.org/packages/6c/16/fdd8930ef4f5795afc3cf89f056645ff012da828a4ddcc6a3d6d41edfb53/qimage2ndarray-1.8.tar.gz (183kB)
     |████████████████████████████████| 184kB 3.5MB/s 
Requirement already satisfied: pandas in ./Library/Python/3.7/lib/python/site-packages (from sleap) (0.25.0)
Requirement already satisfied: pyyaml in /usr/local/lib/python3.7/site-packages (from sleap) (3.13)
Requirement already satisfied: attrs in /usr/local/lib/python3.7/site-packages (from sleap) (19.1.0)
Collecting imgaug==0.3.0
  Downloading https://files.pythonhosted.org/packages/11/df/5a3bba95b4600d5ca7aff072082ef0d9837056dd28cc4e738e7ce88dd8f8/imgaug-0.3.0-py2.py3-none-any.whl (819kB)
     |████████████████████████████████| 829kB 14.7MB/s 
Collecting python-rapidjson
  Downloading https://files.pythonhosted.org/packages/8f/a2/8e7f822889d3fca7e15713ecde25654bef60047f1fce263e3e55e0c233ac/python_rapidjson-0.9.1-cp37-cp37m-macosx_10_6_intel.whl (388kB)
     |████████████████████████████████| 389kB 6.6MB/s 
Collecting jsonpickle==1.2
  Downloading https://files.pythonhosted.org/packages/07/07/c157520a3ebd166c8c24c6ae0ecae7c3968eb4653ff0e5af369bb82f004d/jsonpickle-1.2-py2.py3-none-any.whl
Requirement already satisfied: pyzmq in /usr/local/lib/python3.7/site-packages (from sleap) (18.0.2)
Requirement already satisfied: networkx in ./Library/Python/3.7/lib/python/site-packages (from sleap) (2.3)
Collecting imgstore==0.2.9
  Downloading https://files.pythonhosted.org/packages/39/bb/a1099d1564c1f85752251a9549037ce36021f2b318c896be2f2b067d7e9a/imgstore-0.2.9-py2.py3-none-any.whl (904kB)
     |████████████████████████████████| 911kB 38.8MB/s 
Collecting cattrs==1.0.0rc
  Downloading https://files.pythonhosted.org/packages/01/f6/30e2b78920dae271ba94cb74a808021d88840b156c424c029f6255b72d3e/cattrs-1.0.0rc0-py2.py3-none-any.whl (59kB)
     |████████████████████████████████| 61kB 5.5MB/s 
Collecting PySide2<=5.14.1,>=5.12.0
  Downloading https://files.pythonhosted.org/packages/83/93/102d20680b5110cec805fbc34ad53ee49e325b4a8f10cad6f447c10af709/PySide2-5.14.1-5.14.1-cp35.cp36.cp37.cp38-abi3-macosx_10_13_intel.whl (148.3MB)
     |████████████████████████████████| 148.3MB 354kB/s 
Collecting psutil
  Downloading https://files.pythonhosted.org/packages/c4/b8/3512f0e93e0db23a71d82485ba256071ebef99b227351f0f5540f744af41/psutil-5.7.0.tar.gz (449kB)
     |████████████████████████████████| 450kB 4.1MB/s 
Requirement already satisfied: scikit-learn in ./Library/Python/3.7/lib/python/site-packages (from sleap) (0.21.3)
Requirement already satisfied: scipy>=1.0.1 in ./Library/Python/3.7/lib/python/site-packages (from seaborn->sleap) (1.4.1)
Requirement already satisfied: matplotlib>=2.1.2 in /usr/local/lib/python3.7/site-packages (from seaborn->sleap) (3.1.1)
Requirement already satisfied: numpy>=1.13.3 in ./Library/Python/3.7/lib/python/site-packages (from seaborn->sleap) (1.17.2)
Requirement already satisfied: pillow in ./Library/Python/3.7/lib/python/site-packages (from scikit-video->sleap) (6.1.0)
Requirement already satisfied: google-pasta>=0.1.6 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (0.1.7)
Requirement already satisfied: protobuf>=3.8.0 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (3.9.1)
Requirement already satisfied: gast==0.2.2 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (0.2.2)
Requirement already satisfied: termcolor>=1.1.0 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (1.1.0)
Requirement already satisfied: absl-py>=0.7.0 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (0.7.1)
Requirement already satisfied: wrapt>=1.11.1 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (1.11.2)
Requirement already satisfied: keras-preprocessing>=1.1.0 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (1.1.0)
Requirement already satisfied: astor>=0.6.0 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (0.8.0)
Requirement already satisfied: six>=1.12.0 in /usr/local/lib/python3.7/site-packages (from tensorflow==2.1->sleap) (1.12.0)
Requirement already satisfied: opt-einsum>=2.3.2 in /usr/local/lib/python3.7/site-packages (from tensorflow==2.1->sleap) (3.1.0)
Collecting tensorboard<2.2.0,>=2.1.0
  Using cached https://files.pythonhosted.org/packages/d9/41/bbf49b61370e4f4d245d4c6051dfb6db80cec672605c91b1652ac8cc3d38/tensorboard-2.1.1-py3-none-any.whl
Collecting tensorflow-estimator<2.2.0,>=2.1.0rc0
  Using cached https://files.pythonhosted.org/packages/18/90/b77c328a1304437ab1310b463e533fa7689f4bfc41549593056d812fab8e/tensorflow_estimator-2.1.0-py2.py3-none-any.whl
Requirement already satisfied: grpcio>=1.8.6 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (1.22.0)
Requirement already satisfied: keras-applications>=1.0.8 in ./Library/Python/3.7/lib/python/site-packages (from tensorflow==2.1->sleap) (1.0.8)
Requirement already satisfied: wheel>=0.26; python_version >= "3" in /usr/local/lib/python3.7/site-packages (from tensorflow==2.1->sleap) (0.33.6)
Requirement already satisfied: imageio>=2.0.1 in ./Library/Python/3.7/lib/python/site-packages (from scikit-image->sleap) (2.5.0)
Requirement already satisfied: PyWavelets>=0.4.0 in ./Library/Python/3.7/lib/python/site-packages (from scikit-image->sleap) (1.0.3)
Requirement already satisfied: pytz>=2017.2 in ./Library/Python/3.7/lib/python/site-packages (from pandas->sleap) (2019.2)
Requirement already satisfied: python-dateutil>=2.6.1 in /usr/local/lib/python3.7/site-packages (from pandas->sleap) (2.8.0)
Requirement already satisfied: Shapely in ./Library/Python/3.7/lib/python/site-packages (from imgaug==0.3.0->sleap) (1.6.4.post2)
Requirement already satisfied: decorator>=4.3.0 in /usr/local/lib/python3.7/site-packages (from networkx->sleap) (4.4.0)
Collecting tzlocal
  Downloading https://files.pythonhosted.org/packages/5d/94/d47b0fd5988e6b7059de05720a646a2930920fff247a826f61674d436ba4/tzlocal-2.1-py2.py3-none-any.whl
Collecting shiboken2==5.14.1
  Downloading https://files.pythonhosted.org/packages/63/a0/b3d3b5e501a9f81fd19cf82e2e37c7cd9bd6583068c9f75ffa389bcaa031/shiboken2-5.14.1-5.14.1-cp35.cp36.cp37.cp38-abi3-macosx_10_13_intel.whl (829kB)
     |████████████████████████████████| 829kB 18.7MB/s 
Requirement already satisfied: joblib>=0.11 in ./Library/Python/3.7/lib/python/site-packages (from scikit-learn->sleap) (0.13.2)
Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /usr/local/lib/python3.7/site-packages (from matplotlib>=2.1.2->seaborn->sleap) (2.4.2)
Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.7/site-packages (from matplotlib>=2.1.2->seaborn->sleap) (0.10.0)
Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.7/site-packages (from matplotlib>=2.1.2->seaborn->sleap) (1.1.0)
Requirement already satisfied: setuptools in /usr/local/lib/python3.7/site-packages (from protobuf>=3.8.0->tensorflow==2.1->sleap) (42.0.2)
Requirement already satisfied: markdown>=2.6.8 in ./Library/Python/3.7/lib/python/site-packages (from tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (3.1.1)
Requirement already satisfied: google-auth-oauthlib<0.5,>=0.4.1 in ./Library/Python/3.7/lib/python/site-packages (from tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (0.4.1)
Requirement already satisfied: werkzeug>=0.11.15 in ./Library/Python/3.7/lib/python/site-packages (from tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (0.15.5)
Requirement already satisfied: google-auth<2,>=1.6.3 in ./Library/Python/3.7/lib/python/site-packages (from tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (1.10.1)
Requirement already satisfied: requests<3,>=2.21.0 in /usr/local/lib/python3.7/site-packages (from tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (2.22.0)
Requirement already satisfied: requests-oauthlib>=0.7.0 in ./Library/Python/3.7/lib/python/site-packages (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (1.3.0)
Requirement already satisfied: pyasn1-modules>=0.2.1 in ./Library/Python/3.7/lib/python/site-packages (from google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (0.2.8)
Requirement already satisfied: rsa<4.1,>=3.1.4 in ./Library/Python/3.7/lib/python/site-packages (from google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (4.0)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in ./Library/Python/3.7/lib/python/site-packages (from google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (4.0.0)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/site-packages (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (1.25.6)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.7/site-packages (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (3.0.4)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/site-packages (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (2019.9.11)
Requirement already satisfied: idna<2.9,>=2.5 in /usr/local/lib/python3.7/site-packages (from requests<3,>=2.21.0->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (2.8)
Requirement already satisfied: oauthlib>=3.0.0 in ./Library/Python/3.7/lib/python/site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (3.1.0)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in ./Library/Python/3.7/lib/python/site-packages (from pyasn1-modules>=0.2.1->google-auth<2,>=1.6.3->tensorboard<2.2.0,>=2.1.0->tensorflow==2.1->sleap) (0.4.8)
Building wheels for collected packages: jsmin, qimage2ndarray, psutil
  Building wheel for jsmin (setup.py) ... done
  Created wheel for jsmin: filename=jsmin-2.2.2-cp37-none-any.whl size=13919 sha256=c6a483279cdc560c585100496cc85569c757b9318540330ac3f8680140661687
  Stored in directory: /Users/jake/Library/Caches/pip/wheels/64/f4/de/9667d84f759289edf5442220997c6d4334637a6bb2a7b90f73
  Building wheel for qimage2ndarray (setup.py) ... done
  Created wheel for qimage2ndarray: filename=qimage2ndarray-1.8-cp37-none-any.whl size=11696 sha256=9360c843e17faff4cd2b6c81da8ed64853c9d55ddebf8f88508a65651bb5a424
  Stored in directory: /Users/jake/Library/Caches/pip/wheels/52/ef/1d/c77ef6d3d3fe72aa88f822a05e29b17e79914af69ddeff4c02
  Building wheel for psutil (setup.py) ... done
  Created wheel for psutil: filename=psutil-5.7.0-cp37-cp37m-macosx_10_14_x86_64.whl size=226885 sha256=1c506df23428c2d98eeb7af4c47232ab41572848f2ebe004c788fdd21751db69
  Stored in directory: /Users/jake/Library/Caches/pip/wheels/d7/69/b4/3200b95828d1f0ddb3cb5699083717f4fdbd9b4223d0644c57
Successfully built jsmin qimage2ndarray psutil
ERROR: tensorboard 2.1.1 has requirement grpcio>=1.24.3, but you'll have grpcio 1.22.0 which is incompatible.
Installing collected packages: seaborn, jsmin, scikit-video, tensorboard, tensorflow-estimator, tensorflow, opencv-python-headless, qimage2ndarray, imgaug, python-rapidjson, jsonpickle, tzlocal, imgstore, cattrs, shiboken2, PySide2, psutil, sleap
  Found existing installation: tensorboard 2.0.2
    Uninstalling tensorboard-2.0.2:
      Successfully uninstalled tensorboard-2.0.2
  Found existing installation: tensorflow-estimator 2.0.1
    Uninstalling tensorflow-estimator-2.0.1:
      Successfully uninstalled tensorflow-estimator-2.0.1
  Found existing installation: tensorflow 2.0.1
    Uninstalling tensorflow-2.0.1:
      Successfully uninstalled tensorflow-2.0.1
  Found existing installation: opencv-python-headless 4.1.0.25
    Uninstalling opencv-python-headless-4.1.0.25:
      Successfully uninstalled opencv-python-headless-4.1.0.25
  Found existing installation: imgaug 0.2.9
    Uninstalling imgaug-0.2.9:
      Successfully uninstalled imgaug-0.2.9
Successfully installed PySide2-5.14.1 cattrs-1.0.0rc0 imgaug-0.3.0 imgstore-0.2.9 jsmin-2.2.2 jsonpickle-1.2 opencv-python-headless-4.2.0.34 psutil-5.7.0 python-rapidjson-0.9.1 qimage2ndarray-1.8 scikit-video-1.1.11 seaborn-0.10.1 shiboken2-5.14.1 sleap-1.0.0 tensorboard-2.1.1 tensorflow-2.1.0 tensorflow-estimator-2.1.0 tzlocal-2.1```

**Comment by ntabris** [comment]
Thanks for catching this! Looks like we need attrs 19.3. You should be able to fix it for now with `python3 -m pip install attrs==19.3`

(I'll get this fixed in the PyPI package for SLEAP v1.0.1)

**Comment by ntabris** [comment]
SLEAP v1.0.1 is on PyPI.
